### PR TITLE
Add a new example

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -39,13 +39,30 @@ under the License.
...
 		<!-- Hologres connector -->
 
 		<dependency>
-			<groupId>io.hologres</groupId>
-			<artifactId>hologress-flink-connector</artifactId>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>hologres-flink-connector</artifactId>
 			<version>${hologres.flink.version}</version>
-			<scope>provided</scope>
+			<classifier>jar-with-dependencies</classifier>
```
Comment (bowenli86) [inline pom.xml:65]: are the group/artifact ids in the connector?

File: pom.xml [hunk 2]
```diff
@@ -112,6 +128,25 @@ under the License.
 					<target>${java.version}</target>
 				</configuration>
 			</plugin>
+
+			<plugin>
+				<artifactId>maven-assembly-plugin</artifactId>
```
Comment (bowenli86) [inline pom.xml:133]: why do we need to create a jar for these exmaples?

File: src/main/java/io/hologres/flink/example/HologresSinkExample.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.TableSchema;
+
+public class HologresSinkExample {
```
Comment (bowenli86) [inline src/main/java/io/hologres/flink/example/HologresSinkExample.java:14]: is this the jdbc or rpc sink? can we specify that in the class name?

File: pom.xml (hunk 1 above)
Comment (huafengw) [reply pom.xml:65]: I think hologram/flink-connector-hologres has already changed to this group ID and artifactId

File: pom.xml (hunk 2 above)
Comment (huafengw) [reply pom.xml:133]: This demo is intended to be submitted to a user cluster for testing.

File: src/main/java/io/hologres/flink/example/HologresSinkExample.java (hunk 3 above)
Comment (huafengw) [reply src/main/java/io/hologres/flink/example/HologresSinkExample.java:14]: I think we should hide these implementation details

**Pull request merged**